chain/network/src/types.rs

```diff
@@ -525,27 +528,155 @@ impl Message for PeerRequest {
     type Result = PeerResponse;
 }
 
-#[derive(MessageResponse)]
+#[derive(MessageResponse, Debug)]
 pub enum PeerResponse {
     NoResponse,
     UpdatedEdge(EdgeInfo),
 }
 
 /// Requesting peers from peer manager to communicate to a peer.
-#[derive(Clone)]
+#[derive(Clone, Debug)]
 pub struct PeersRequest {}
 
 impl Message for PeersRequest {
     type Result = PeerRequestResult;
 }
 
 /// Received new peers from another peer.
-#[derive(Message)]
+#[derive(Message, Debug, Clone)]
 #[rtype(result = "()")]
 pub struct PeersResponse {
     pub peers: Vec<PeerInfo>,
 }
 
+#[derive(Debug)]
+pub enum PeerMessageRequest {
```

Comment:
Can you add comment describing this enum (what it is used for etc)

Reply:
Done